### ISSUE TITLE
Is it compatible for rails 4?

### ISSUE THREAD

**Issue by avellable** [Octo-Labs/easymarklet#4]
I keep getting 

Could not find generator easymarklet.

Is easymarkelt compatible with rails 4. I checked and the gem is installed properly.

**Comment by xxx** [comment]
This gem is _not_ compatible with Rails 4 out of the box, but it can be massaged to work. I've not had a problem with the generator, but was just bitten by the fact that Rails 4 sets the X-Frame-Options header by default now.

**Comment by jagthedrummer** [comment]
Hey @avellable, I just created a new rails 4 app, added `gem 'easymarklet'` to the `Gemfile`, ran `bundle install`, and then successfully ran `rails generate easymarklet:simple foobaz`.

@xxx Have you had problems other than the X-Frame-Options issue, or is that the only one? I'll add some documentation about how to get around that.

**Comment by xxx** [comment]
@jagthedrummer There are also issues around Rails 4's removal of non-digest assets which we're working around now. Beyond that, things seem to be working.

**Comment by jagthedrummer** [comment]
Thanks for the info, @xxx. Any chance you could share what you did to work around the asset issue? I'd love to include either an implemented workaround or instructions on how to do it. Cheers!

**Comment by xxx** [comment]
@jagthedrummer I'm only contracting on the project, so I don't have authority to give out what's being done, but paging @sumitngupta , who does.

**Comment by sumitngupta** [comment]
Hey @jagthedrummer. I'm still presently wrestling a bit with getting our bookmarklet served properly. I'm using https://github.com/alexspeller/non-stupid-digest-assets to get the custom assets we've built to be served properly without a digest key, and now trying to figure how why the easyXDM.js library is being served with a non-ssl url.  Presently I'm pulled off on working on something else but hoping to be able to spend some time on it this weekend. I will update with the full set of the steps I had to do (assuming I get it to work before I smash my computer).

**Comment by xxx** [comment]
@sumitngupta Is there any reason why protocol-relative urls aren't being used?

**Comment by sumitngupta** [comment]
@xxx all of our custom js/css are being served properly under SSL, I got to the point where I saw that the easyXDM.js library is being served with a non-secure protocol and got pulled off onto other stuff. Im hoping its something trivial but won't know more till I dive back in. easyXDM.js lives inside the easymarklet gem and not in one of our asset folders.. I'm not sure how that plays in since I thought all requests to assets from my system are getting converted to SSL regardless. Its been a game of whack-a-mole with browser security restrictions. 

Just for recording something I've already tried: https://github.com/sikachu/sprockets-redirect does _not_ solve any of these problems, it actually caused a lot more security warnings (as expected since its causing redirects).
